user_update.php

```diff
@@ -0,0 +1,22 @@
+<?php
+require 'db_connection.php';
+// --------------------------------------------------
+// パラメータを取得
+// --------------------------------------------------
+$id = $_POST['id'];
+$password = $_POST['pass'];
+$admin_flag = $_POST['admin_flag'];
+
+// --------------------------------------------------
+// ユーザーを更新
+// --------------------------------------------------
+if (isset($db)) {
+    $sql = 'update users set password = :password, admin_flag = :admin_flag, updated_at = current_timestamp() where id = :id';
+    $stmt = $db->prepare($sql);
+    $params = array(':password' => $password, ':admin_flag' => $admin_flag, ':id' => $id);
```

Comment:
これでも何の問題もないんですが
```suggestion
    $params = [':password' => $password, ':admin_flag' => $admin_flag, ':id' => $id];
```
PHPのバージョンによっては配列はこう書けたりいます。

Reply:
初期化も
```
$a = array(); → $a = [];
```
でいけたりします。